tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/workspace/NewLiferayWorkspaceOp.java

```diff
@@ -68,7 +68,7 @@
     @AbsolutePath
     @Enablement( expr = "${ UseDefaultLocation == 'false' }" )
     @ValidFileSystemResourceType( FileSystemResourceType.FOLDER )
-    @Label( standard = "location" )
```

Comment:
This doesn't need to change for the normal New Workspace Op.  This field label should only change in Studio.